.github/workflows/npm_publish.yml

```diff
@@ -24,16 +24,6 @@ jobs:
 
       - name: Publish npm package to both places
         run: |
-          yarn publish --access public
-          sed -i -e 's#"@stellar/stellar-base"#"stellar-base"#' package.json
-          yarn publish
-        env:
-          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
-
-      - name: Deprecate the old package
-        run: |
-          V=$(cat package.json | jq '.version' | sed -e 's/\"//g')
-          echo "Deprecating stellar-base@$V"
-          npm deprecate stellar-base@"<= $V" "⚠️ This package has moved to @stellar/stellar-base! 🚚"
+          yarn publish --access public --tag protocol-23
```

Comment:
maybe  `--tag beta` as that seems to be what is being implied, since once it goes to ga, then this same dist won't have `--tag protocol-23` anymore right?

Reply:
We had this conversation last time (https://github.com/stellar/js-stellar-base/pull/779#discussion_r1775597167) :laughing: I'll change it to `protocol-23-beta` to match!